doc/docs/master/concepts/pipeline-concepts/pipeline/spout.md

```diff
@@ -83,3 +83,18 @@ The following text is an example of a minimum specification:
   }
 }
 ```
+
+When a spout container crashes, all changes that were
+processed before the crash are lost, and the spout needs
+to start the interrupted data operation from scratch.
+To keep the history of changes, so that the spout can
+continue where it left off after the restart, you can
+enable a `marker` for your spout.
+
+When you specify the `marker` parameter in the
+`transform` section of your pipeline, Pachyderm creates
+the `marker` file or directory. The current spout marker
+is stored in `pfs/out/marker`. The previous marker is stored
+in `pfs/marker`. If a spout container crashes and then starts
```

Comment:
The _current_ marker is in `pfs/out/marker` and the previous one is in `pfs/marker`? Not the other way around?

Reply:
@adelelopez can you clarify this one?

Reply:
That's right